packages/react-native/Libraries/Components/Switch/Switch.js

```diff
@@ -213,7 +213,7 @@ const Switch: component(
         SwitchCommands.setValue(nativeSwitchRef.current, jsValue);
       }
     }
-  }, [value, native]);
+  }, [value, native.value]);
```

Comment:
This isn't safe. We rely on the object identity of native changing to correctly implement Switch as a controlled component. Fixing in https://github.com/facebook/react-native/pull/50928

Reply:
> This isn't safe. We rely on the object identity of native changing to correctly implement Switch as a controlled component. Fixing in #50928

I apologize for the code change. I wasn't aware that it relied on the object identity of `native` changing. Sorry for the confusion.

Reply:
All good. I, as the reviewer, also wasn’t aware. 😅